server/tso/tso.go

```diff
@@ -44,6 +44,10 @@ const (
 	maxLogical = int64(1 << 18)
 	// MaxSuffixBits indicates the max number of suffix bits.
 	MaxSuffixBits = 4
+	// jetLagWarningThreshold is the warning threshold of jetLag in `timestampOracle.UpdateTimestamp`.
+	// In case of small `updatePhysicalInterval`, the `3 * updatePhysicalInterval` would also is small,
+	// and trigger many un-necessary warnings about clock offset.
+	jetLagWarningThreshold = 30 * time.Millisecond
```

Comment:
Why the value is 30ms?

Reply:
It's not strictly calculated, but an empirical value.

On benchmark with `updatePhhsicalInterval = 1ms`, I observed a frequent log about jet lag ("clock offset ..."), saying a `jet-lag` varying between `5ms ~ 30ms`. I think it is not caused by a clock offset but other reasons (such as go runtime scheduling duration).

And I also think `30ms` would be a reasonable go routine wait duration on a busy system. So I choose such a value.

Reply:
yes， I think the original value is 150ms (3 *minTSOUpdatePhysicalInterval(50ms) ). I think the 30ms maybe too small .

Reply:
Fixed, PTAL~